src/create_disk.sh

```diff
@@ -477,22 +481,10 @@ generate_gpgkeys() {
 case "$arch" in
 x86_64)
     # UEFI
```

Comment:
```suggestion
    # bootupd does UEFI and BIOS now
```

or I guess we could just delete the comment